### ISSUE TITLE
Restating flash and red flash thresholds for clarity

### ISSUE THREAD

**Issue by awkawk** [w3c/wcag#69]
http://lists.w3.org/Archives/Public/public-comments-wcag20/2015Jan/0009.html

We are reviewing and modifying our Japanese  translation of WCAG 2 for JIS refresh.

We have a request on "general flash and red flash thresholds" in "Appendix A: Glossary".

It reads "Note 2: A transition is the change in relative luminance (or relative luminance/color for red flashing) between adjacent peaks and valleys in a plot of relative luminance (or relative luminance/color for red flashing) measurement against time."

Could you restate Note 2 in different words (in plain English)?

**Comment by joshueoconnor** [comment]
https://www.w3.org/2006/02/lc-comments-tracker/35422/NOTE-UNDERSTANDING-WCAG20-20140911/2991

**Comment by marcjohlic** [comment]
Propose dropping "Note 2" completely and substituting the word "changes" for "transitions" in the "general flash and red flash thresholds" definition.

The second bullet point under "where:" would change to:

* A **red flash** is defined as any pair of opposing @@changes involving a saturated red.

Rationale for these edits:

Note 2 seems to only define a "transition".  However, the original definition of "flash" and "general flash" use forms of the word "change" (changing, changes etc).  Therefore, I propose that "changes" should also be used for the "red flash" definition in place of "transitions".

Further, HardingPFA.com states:  ".. red flash is where: There are opposing **changes** to or from saturated red colour ..".  Previous work by Harding in this area was referenced when this appendix was first put together (Note 3 of the same term).

FWIW, Note 2 does make sense after you read it a few times.  It's actually quite simple and I don't know that it could be wordsmithed much more and keep the same point.  But to the commentor's point, it is overly verbose - due largely in part to the parenthetical content - and therefore confusing.

**Comment by joshueoconnor** [comment]
Great stuff Marc.

**Comment by marcjohlic** [comment]
**Proposed Resolution**
After reviewing "Note 2", the Working Group agrees that it can be removed altogether and the bullet point preceding it under "Where:" will be changed to:

* A **red flash** is defined as any pair of opposing changes involving a saturated red.

**Comment by awkawk** [comment]
This is in the Normative glossary.  We can add it to the "future tasks" list but we can't change it.

**Comment by marcjohlic** [comment]
In an effort to clarify Note 2 and define "transition":  

"Where colors are changing on the screen, the transition is the change between the highest relative luminance and the lowest relative luminance."

**Comment by awkawk** [comment]
How about:

http://www.w3.org/TR/WCAG20/#general-thresholddef 
Note 2:
A transition is the change from a maximum relative luminance value to a minimum relative luminance value (or minimum to maximum). A flash consists of two opposing transitions.

**Comment by awkawk** [comment]
I'll add that the current definition doesn't include the word 'color' so I'm avoiding it also.  We've run into difficulties with this with contrast where we say "color luminosity" rather than just "color".  The luminosity might change because of the hue (often thought of as color) but it might change because of a difference in the saturation or value, etc.

**Comment by marcjohlic** [comment]
**Proposed Resolution**
Thank you for your request.  This content is in the Normative glossary and cannot be changed, however we can add a request for clarification of the wording to out "future tasks".

In an effort to help clarify the wording for the purpose of your translation work, the Working Group agreed on this alternate wording for Note 2:

Note 2:
A transition is the change from a maximum relative luminance value to a minimum relative luminance value (or minimum to maximum). A flash consists of two opposing transitions.

**Comment by awkawk** [comment]
Gregg's proposal:
Thank you Makoto.   Here is what that Note is saying: 
  
If you plot the relative luminance over time - you will see peaks and valleys in the plot.   A transition is the change in value from one Peak to the next valley (or from one valley to the next peak) on the plot.   A flash is two opposing transitions.   
Opposing transitions means  transitions going in the opposite direction.  
for example -     a transition from a peak to a valley followed and from that valley to the next peak would be a flash
a transition from valley to next peak to next valley   would also be a flash. 
If either of these happens 3 times in a second    valley-peak-valley-peak-valley-peak-valley   OR   peak-valley-peak-valley-peak-valley-peak     in any 1 second period - it would fail. 
For red flashing you would use a plot of the “relative luminance/color”.
  
  
(We have also added this to the Understanding WCAG 2.0 document)

**Comment by joshueoconnor** [comment]
If you plot the relative luminance over time - you will see peaks and valleys in the plot. A transition is the change in value from one Peak to the next valley (or from one valley to the next peak) on the plot. A flash is two opposing transitions.

<del>Opposing transitions means transitions going in the opposite direction.</del>

For example - a transition from a peak to a valley followed from that valley to the next peak would be a flash a transition from valley to next peak to next valley would also be a flash. 
If either of these happens 3 times in a second valley-peak-valley-peak-valley-peak-valley OR peak-valley-peak-valley-peak-valley-peak in any 1 second period - it would fail. 
For red flashing you would use a plot of the “relative luminance/color”.

**Comment by awkawk** [comment]
The group approves the wording in general and Makoto will bring it to his committee.

A WCAG editor will integrate this text with some minor editorial changes into the Understanding document and submit a pull request.

**Comment by awkawk** [comment]
So, we have a problem with this and I'd like some additional thoughts.  

We are proposing adding the additional description from Gregg to the understanding document, but the additional text is about a definition and there is no section in the understanding document that applies to the glossary.

We could add it to the 2.3.1 section (http://www.w3.org/TR/UNDERSTANDING-WCAG20/complete.html#seizure-does-not-violate) but it doesn't seem to fit.

Ideas?

**Comment by michael-n-cooper** [comment]
I'm taking a hasty look at this but think it might be ok to add a 
sub-section with a heading in the Understanding for the relevant SC 
saying "note on the definition of @@". We don't have a precedent for it 
afaik but we don't rule it out, and if it aids understanding it would be 
useful to do. Michael

**Comment by michael-n-cooper** [comment]
Doing issue walk through this appears to be addressed closing
